src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -99,10 +99,23 @@ private void RootCommandHandler(FileInfo templateFilePath, FileInfo parametersFi
             }
             catch (Exception exp)
             {
-                Console.WriteLine($"An exception occured: {exp.Message}");
+                Console.WriteLine($"An exception occured: {GetAllExceptionMessages(exp)}");
             }
         }
 
+        private static string GetAllExceptionMessages(Exception exception)
+        {
+            string exceptionMessage = exception.Message;
+
+            while (exception.InnerException != null)
+            {
+                exception = exception.InnerException;
+                exceptionMessage += " " + exception.Message;
```

Comment:
Can we use a different separating character besides a single space?

Reply:
@JohnathonMohr Any suggestions? I can add a new line or maybe "Inner Exception: ".

Reply:
Maybe a "|" pipe character?  Or surround messages in square brackets?  The issue with a space is it can make distinguishing distinct messages difficult, like if a message doesn't end with a period (or has multiple sentences with periods) - where one ends and other starts could be ambiguous and make the overall message hard to interpret.

As long as we know messages are easy to distinguish, *how* they're separated isn't terribly important.  It just needs to be *easy* :)